### PR TITLE
Update; include sub-recipe for snpEff 4.5covid19

### PR DESCRIPTION
**Pull request by tomkinsc**
A special build of snpEff has been published that comes pre-packaged with the reference for SARS-CoV-2. This adds a sub-recipe for that build of snpEff, while leaving the main recipe unchanged. The main recipe should be updated once the primary release of snpEff has been updated to v4.5.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/snpeff/4.5covid19/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  noarch: generic
+
+source:
+  url: https://downloads.sourceforge.net/project/snpeff/snpEff_{{ snpeff_ver }}_core.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  run:
+    - openjdk
+    - zlib
```
Comment (bgruening) [inline recipes/snpeff/4.5covid19/meta.yaml:26]: zlib can be removed

File: recipes/snpeff/4.5covid19/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+{% set version = "4.5covid19" %}
+{% set sha256 = "d4978b4ed51edba8458c8822bc6271794ef1f36b467d3942dbb56c7d5ae494ea" %}
+
+about:
+  home: 'http://snpeff.sourceforge.net/'
+  license: "LGPLv3"
+  summary: "Genetic variant annotation and effect prediction toolbox"
+
+package:
+  name: snpeff
```
Comment (bgruening) [inline recipes/snpeff/4.5covid19/meta.yaml:12]: if we just change the version, why not changing then the original recipe?

Comment (tomkinsc) [reply recipes/snpeff/4.5covid19/meta.yaml:12]: We can, I suppose, though I assume there is a reason the author does not want this release to be the official one as it has been out for three weeks but it is not (yet) the main release on [GitHub](https://github.com/pcingola/SnpEff/releases) or [SourceForge](http://snpeff.sourceforge.net/).

Comment (bgruening) [reply recipes/snpeff/4.5covid19/meta.yaml:12]: mh, ok strange

**Comment by tomkinsc** [comment]
@BiocondaBot please merge

**Pull request merged**